src/components/Buttons/Button/Button.tsx

```diff
@@ -7,19 +7,19 @@ import { ButtonType, Context, Size } from '../../../constants';
 // supported although not defined in propTypes
 interface Props extends Omit<React.ButtonHTMLAttributes<HTMLButtonElement>, 'size'> {
     /** The label of the button */
-    children: string | React.ReactNode;
+    children: string | React.ReactElement;
     /** The button context (e.g. brand, primary, bad, good etc. - defaults to neutral) */
-    context: Context | 'link';
+    context?: Context | 'link';
     /** The size of the button */
-    size: Size;
+    size?: Size;
     /** Whether or not to show block-level button (full width) */
-    isBlock: boolean;
+    isBlock?: boolean;
     /** Whether or not to show inline button (without padding) */
-    isInline: boolean;
+    isInline?: boolean;
     /** Should button be disabled or not */
-    disabled: boolean;
+    disabled?: boolean;
     /** Type of the button */
-    type: ButtonType;
+    type?: ButtonType;
     /** Providing an href will render an <a> element, styled as a button. */
     href?: string;
 }
```

Comment:
seems you forgot to move to typeScript `Button/index.js`, `Callout/index.js`, `CandidateAvatar/index.ts`

Reply:
Yes, they were converted in a different PR, but I can approve on it...